app/Http/Controllers/Admin/PermissionGroupController.php

```diff
@@ -0,0 +1,105 @@
+<?php
+
+namespace App\Http\Controllers\Admin;
+
+use App\Http\Controllers\Controller;
+use App\Http\Requests\Admin\PermissionGroupRequest;
+use App\Repositories\Admin\PermissionGroup\PermissionGroupInterface;
+
+class PermissionGroupController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public $permissionGroupRepository;
+
+    public function __construct(PermissionGroupInterface $permissionGroupRepository)
+    {
+        $this->permissionGroupRepository  = $permissionGroupRepository;
+    }
+
+    public function index()
+    {
+        $db = $this->permissionGroupRepository->paginate();
+        return view('admin.permission.permissiongroup.index', compact('db'));
```

Comment:
return view('admin.permission.permissiongroup.index', compact('db'));
=>

return view('admin.permission.permissiongroup.index', [
         'permissionGroups' => $this->permissionGroupRepository->paginate()
]);